.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,36 @@
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+| Name| about |title|labels|
+| ------------- | ------------- |------------- |------------- |
+| Feature Request  | Suggest an idea for this project  |[Short Description] (Version)|enhancement|
+
+**Read before creating an issue**
+In general, we close any issues that are
```

Comment:
Can you revise this part as shown below?

In general, we close any issues that are

- unactionable (fill the template below under "Description")
- questions best served elsewhere

Reply:
I cannot see any changes in this part. Can you revise it again?